test/api/controllers/routerController/routeMQListener.test.js

```diff
@@ -6,6 +6,7 @@
 var
   should = require('should'),
   winston = require('winston'),
+  q = require('q'),
   params = require('rc')('kuzzle'),
   Kuzzle = require('root-require')('lib/api/Kuzzle'),
```

Comment:
root-require is in the devDependencies part of the package.json file, as it is only used for unit testing.

However it seems that we can indeed remove this by using require.main. I'll prepare another pull requests.